### PR TITLE
Small tweaks to title algorithm

### PR DESCRIPTION
**Pull request by edwardhorsford**
Some small tweaks to the allegation title algorithm:

* Use sentence case rather than title case
* append 'hazard' to hazard type
* Include name 'Allegation' for edge case where a case has no products and no product category.

### Before:
**Electrical appliances and equipment – Fire (no product specified)**
### After:
**Electrical appliances and equipment – fire hazard (no product specified)**
### Before:
**Electric 546 Shaver, Shaver – Other**
### After:
**Electric 546 Shaver, Shaver – other hazard**

### CODE REVIEW

File: psd-web/app/decorators/investigation/allegation_decorator.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Investigation < ApplicationRecord
   class AllegationDecorator < InvestigationDecorator
     def title
-      title = build_title_from_products || ""
-      title << " – #{object.hazard_type}" if object.hazard_type.present?
+      title = build_title_from_products || "Allegation"
```
Comment (frankieroberto) [inline psd-web/app/decorators/investigation/allegation_decorator.rb]: Add a test for this edge case where there's no product/category?

File: psd-web/app/decorators/investigation/allegation_decorator.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+      title << " – #{object.hazard_type.downcase} hazard" if object.hazard_type.present?
```
Comment (frankieroberto) [inline psd-web/app/decorators/investigation/allegation_decorator.rb]: Is there a finite list of `hazard_type`s? If so, it might be worth quickly checking that they can all be safely lowercased (ie there's no Proper Nouns in there)."

File: psd-web/spec/decorators/investigation/allegation_decorator_spec.rb [hunk 3]
```diff
@@ -40,8 +40,8 @@
     context "when no products are present on the case" do
       let(:allegation) { investigations :no_products_case_title }
 
-      it "has the corect title" do
-        expect(subject.title).to eq("Alarms – Asphyxiation (no product specified)")
+      it "has the correct title" do
```
Comment (frankieroberto) [inline psd-web/spec/decorators/investigation/allegation_decorator_spec.rb]: 😂

File: psd-web/app/decorators/investigation/allegation_decorator.rb (hunk 2 above)
Comment (edwardhorsford) [reply psd-web/app/decorators/investigation/allegation_decorator.rb]: There are no proper nouns in the list.

File: psd-web/app/decorators/investigation/allegation_decorator.rb (hunk 1 above)
Comment (edwardhorsford) [reply psd-web/app/decorators/investigation/allegation_decorator.rb]: I'd need someone else to help with that.
